src/components/Checkbox.js

```diff
@@ -0,0 +1,58 @@
+import React, { useState, useEffect } from 'react';
+
+import { db } from '../lib/firebase';
+
+const Checkbox = ({ item, shoppingList }) => {
+  const [checked, setChecked] = useState(false);
+
+  const { name, id, lastPurchasedDate } = item;
+
+  const isExpired = (date) => {
+    const expiryDate = item.lastPurchasedDate + 60 * 60 * 24;
+    const now = new Date().getTime();
+    return expiryDate < now;
+  };
+
+  useEffect(() => {
+    if (lastPurchasedDate === null) {
+      setChecked(false);
+    }
+    if (isExpired(lastPurchasedDate)) {
+      setChecked(false);
+    } else {
+      setChecked(true);
+    }
+  }, [lastPurchasedDate]);
+
+  const tickCheckBox = async (e) => {
+    if (e.target.checked) {
```

Comment:
What if the user wants to uncheck? Isn't the user supposed to have an option to uncheck whenever they want?

Reply:
I just saw a comment from Julia that the user should be able to uncheck.